### PR TITLE
Improve storybook UX

### PR DESCRIPTION
**Pull request by mayank99**
Added storybook addon-essentials, replacing individual addons for docs, controls and actions.

Includes new viewport and background switcher.
toolbar

Renamed Story tab to Code.
image

Changed Docs tab to respect OS theme, and updated background color for non-inline stories.

### CODE REVIEW

File: .storybook/itwinTheme.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const darkTheme = create({
 
   // UI
   appBg: '#2D373C',
-  appContentBg: '#4F5D65',
+  appContentBg: 'rgb(90, 105, 115)', // background-1
```
Comment (veekeys) [inline .storybook/itwinTheme.js:54]: hex would look better....

Comment (veekeys) [reply .storybook/itwinTheme.js:54]: Didnt need to change for both themes?

Comment (mayank99) [reply .storybook/itwinTheme.js:54]: This is for Docs tab. Previous background looked really bad in Dark theme, but was ok in Light theme.

**Review by veekeys — APPROVED** [review]

File: .storybook/itwinTheme.js (hunk 1 above)
Comment (veekeys) [reply .storybook/itwinTheme.js]: Its not only for docs..its for main too, from what I remember

Comment (mayank99) [reply .storybook/itwinTheme.js]: Yeah but it doesn't look that bad in main tab.

Light theme:
image

Dark theme Docs (current):
image

Dark theme Docs (previous) notice lack of contrast:
image

Dark theme Canvas (current):
image

Dark theme Canvas (previous):
image

**Pull request merged**